yt_dlp/extractor/gedidigital.py

```diff
@@ -1,198 +1,104 @@
-import re
+import json
 
 from .common import InfoExtractor
-from ..utils import (
-    base_url,
-    determine_ext,
-    int_or_none,
-    url_basename,
-    urljoin,
-)
+from ..utils import int_or_none, js_to_json
 
 
 class GediDigitalIE(InfoExtractor):
-    _VALID_URL = r'''(?x:(?P<base_url>(?:https?:)//video\.
+    _VALID_URL = r'''(?x:(?:https?:)//(?:\w+\.)?
         (?:
-            (?:
-                (?:espresso\.)?repubblica
-                |lastampa
-                |ilsecoloxix
-                |huffingtonpost
-            )|
-            (?:
-                iltirreno
-                |messaggeroveneto
-                |ilpiccolo
-                |gazzettadimantova
-                |mattinopadova
-                |laprovinciapavese
-                |tribunatreviso
-                |nuovavenezia
-                |gazzettadimodena
-                |lanuovaferrara
-                |corrierealpi
-                |lasentinella
-            )\.gelocal
```

Comment:
Please explain why these are being removed. Do these gelocal domains no longer exist?

Reply:
Some of them do not exist at all, others (I've found only one atm) started using completely different methods. I'll check them again to see which ones moved to new domains and use the same method